tests/locations/test_package.py

```diff
@@ -1370,6 +1370,89 @@ def test_move_uncompressed_aip(self):
         assert pkg.current_location == dst_location
 
 
+@pytest.fixture
+@pytest.mark.django_db
+def space(tmp_path):
+    space_dir = tmp_path / "space"
+    space_dir.mkdir()
+
+    staging_dir = tmp_path / "staging"
+    staging_dir.mkdir()
+
+    space = models.Space.objects.create(
+        uuid=uuid.uuid4(),
+        access_protocol=models.Space.LOCAL_FILESYSTEM,
+        path=space_dir,
+        staging_path=staging_dir,
+    )
+    models.LocalFilesystem.objects.create(space=space)
+    return space
+
+
+@pytest.fixture
+@pytest.mark.django_db
+def location(space):
+    aipstore = models.Location.objects.create(
+        uuid=uuid.uuid4(),
+        space=space,
+        relative_path="fs-aips",
+        purpose="AS",
+    )
+    models.Location.objects.create(
+        space=space, purpose=models.Location.STORAGE_SERVICE_INTERNAL, relative_path=""
+    )
+    return aipstore
+
+
+@pytest.fixture
+@pytest.mark.django
+def bag_fixture(tmp_path):
+    tmp_dir = tmp_path / "dir"
+    tmp_dir.mkdir()
+    src = os.path.join(FIXTURES_DIR, "working_bag.zip")
+    dst = os.path.join(tmp_dir, "")
+    return shutil.copy(src, dst)
+
+
+@pytest.fixture
+@pytest.mark.django_db
+def package(location, bag_fixture):
+    return models.Package.objects.create(
+        uuid=uuid.uuid4(),
+        current_location=location,
+        current_path=bag_fixture,
```

Comment:
AFAIK the `current_path` of a package is a relative path inside its location. I think you're setting an absolute path here. The path of the package should depend on its location and space. See the [`full_path` property](https://github.com/artefactual/archivematica-storage-service/blob/f4ff4f3ac4bfeb4ba1a2ed895b76189a7f57c408/storage_service/locations/models/package.py#L209-L216)